charts/incubator/ai-server/values.yaml

```diff
@@ -0,0 +1,43 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: codeproject/ai-server
+  tag: cpu-2.5.1@sha256:4ac7e0ace01abdb26e6d1876011a95f9f294352abd23936e4c6040f23582ed9f
+gpu12Image:
+  pullPolicy: IfNotPresent
+  repository: codeproject/ai-server
+  tag: cuda12_2-2.5.1@sha256:c846c67adb140912ff344a7ba57b3498019d18dfeaf70a32ab68ff389db74efd
+gpu11Image:
+  pullPolicy: IfNotPresent
+  repository: codeproject/ai-server
+  tag: cuda11_7-2.5.1@sha256:b48a17af31d10b839baaa07495449bb185493e0d90d88520bd6d6339e6a168cf
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          imageSelector: image
+
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+
+persistence:
+  data:
+    enabled: true
+    mountPath: /etc/codeproject/ai
+  downloads:
+    enabled: true
+    mountPath: /app/modules
+portal:
+  open:
+    enabled: true
+service:
+  main:
+    ports:
+      main:
+        port: 32168
+        targetPort: 32168
```

Comment:
```suggestion

securityContext:
  container:
    runAsNonRoot: false
    readOnlyRootFilesystem: false
    runAsUser: 0
    runAsGroup: 0

service:
  main:
    ports:
      main:
        port: 32168
        targetPort: 32168
        
workload:
  main:
    podSpec:
      containers:
        main:
          imageSelector: image

persistence:
  data:
    enabled: true
    mountPath: /etc/codeproject/ai
  downloads:
    enabled: true
    mountPath: /app/modules
   
portal:
  open:
    enabled: true

```

Reply:
> 

That was the style of the original container, but yeah it's neater